src/stubs.ts

```diff
@@ -62,7 +86,7 @@ export const createStub = ({
   })
 }
 
-const createTransitionStub = ({ name }: StubOptions) => {
+const createTransitionStub = ({ name, propsDeclaration }: StubOptions) => {
```

Comment:
Are the changes to createTransitionStub necessary? I fail to see the link. If it is necessary maybe add a test case with a transition?

Reply:
reverted